src/lib/psk.js

```diff
@@ -0,0 +1,91 @@
+'use strict'
+
+const Details = require('../utils/details')
+const Transport = require('./transport')
+const cryptoHelper = require('../utils/crypto')
+const assert = require('assert')
+
+/**
+  * @module PSK
+  */
+
+/**
+  * Create a payment request using a Pre-Shared Key (PSK).
+  *
+  * @param {Object} params Parameters for creating payment request
+  * @param {String} params.destinationAmount Amount that should arrive in the recipient's account. This value is a string representation of an integer, expressed in the lowest indivisible unit supported by the ledger.
+  * @param {String} params.destinationAccount Target account's ILP address
+  * @param {String} params.sharedSecret Shared secret for PSK protocol
+  * @param {String} [params.id=uuid.v4()] Unique ID for the request (used to ensure conditions are unique per request)
+  * @param {String} [params.expiresAt=30 seconds from now] Expiry of request
```

Comment:
What do you think about making the default for PSK later? In some sense there's no reason to have an expiry at all..